Recrutify/Recrutify.Services/Events/Abstract/IUpdateStatusEventPublisher.cs

```diff
@@ -5,8 +5,12 @@ namespace Recrutify.Services.Events.Abstract
 {
     public interface IUpdateStatusEventPublisher
     {
-        event Func<UpdateStatusEventArgs, Task> StatusCompleted;
+        public event Func<UpdateStatusEventArgs, Task> StatusCompleted;
```

Comment:
In interface we don't write access modifiers, remove public words

Reply:
+